Lib/test/test_ast/test_ast.py

```diff
@@ -814,6 +814,17 @@ def test_repr_large_input_crash(self):
                                     r"Exceeds the limit \(\d+ digits\)"):
             repr(ast.Constant(value=eval(source)))
 
+    def test_unparse_interactive(self):
```

Comment:
This should go in `Lib/test/test_unparse.py`. I also feel like more tests are needed, you could perhaps reuse some from `test_unparse`?